src/Uno.UI/UI/Xaml/UIElement.TextHelper.wasm.cs

```diff
@@ -10,6 +10,7 @@
 using Uno.Disposables;
 using Uno.Extensions;
 using Uno.UI.Xaml;
+using Uno.UI.Xaml.Media;
```

Comment:
```suggestion
```

Reply:
Turns out it has a purpose here, the namespace of ImageDataKind changed 😀